src/Core__Array.res

```diff
@@ -11,6 +13,28 @@ external fromArrayLikeWithMap: (Js.Array2.array_like<'a>, 'a => 'b) => array<'a>
 @val external fromIterator: Core__Iterator.t<'a> => array<'a> = "Array.from"
 @val external fromIteratorWithMap: (Core__Iterator.t<'a>, 'a => 'c) => array<'a> = "Array.from"
 
+let make = (len, x) =>
+  if len <= 0 {
+    []
+  } else {
+    let arr = makeUninitializedUnsafe(len)
+    for i in 0 to len - 1 {
+      arr->setUnsafe(i, x)
+    }
+    arr
+  }
+
+let init = (len, f) =>
```

Comment:
As this creates a new array, too, the function name should start with `make` IMHO, maybe `makeWithInitializer` or something like that.

Reply:
Agree, for the consistency.

Reply:
Or perhaps `fromInitializer`?

Reply:
`fromInitializer` sounds good to me, too.

Reply:
✅ `fromInitializer` it is.

Reply:
Changed in https://github.com/rescript-association/rescript-core/pull/49/commits/889e993f80fd765776fd414cec6bcd69eb4fb4b0